charts/redpanda/values.yaml

```diff
@@ -925,7 +925,8 @@ config:
     # wait_for_leader_timeout_ms: 5000ms                           # Timeout (ms) to wait for leadership in metadata cache
   # -- Node (broker) properties.
   # See the [property reference documentation](https://docs.redpanda.com/docs/reference/node-properties/).
-  node: {}
+  node:
+    crash_loop_limit: 5
```

Comment:
Can you please add a comment explaining where 5 comes from?

Reply:
Great question. I asked @piyushredpanda for core guidance. I found only slack conversation about the default 

https://github.com/redpanda-data/helm-charts/issues/717#issuecomment-1715218308

P.S. Found yet another slack message https://redpandadata.slack.com/archives/C04B5T58QQ1/p1694554270861379?thread_ts=1694549537.193709&cid=C04B5T58QQ1

Reply:
I would like to see good description in core documentation that I could copy-paste it here. At the moment I need to invent the reasoning.

Reply:
This is more art than science. 5 "feels" enough unclean restarts. Not sure if there can be more to it. Agreed that it is better to have a comment on this somewhere and we should document (unsure if it was), @bharathv ?

Reply:
I'd be satisfied with, "5 is chosen with the guidance of the Redpanda core team"

Reply:
> This is more art than science. 5 "feels" enough unclean restarts. Not sure if there can be more to it. Agreed that it is better to have a comment on this somewhere and we should document (unsure if it was), @bharathv ?

Agree, the thought process here is if a broker crashed or shutdown uncleanly for 5 times back to back, something is off and needs a manual intervention.